src/redux/reducers/authReducers.js

```diff
@@ -30,14 +29,13 @@ export default (state = initialState, action) => {
         loading: false,
       };
     case AUTHENTICATION_FAILED:
```

Comment:
When authentication fails, does this mean we don't change state?

Reply:
Yes we are still changing the state same way as before,,

Reply:
Same as logging out,,just tied these two together

Reply:
Ahh. Missed that. Cool